docs/tasks/stateful-sets/deleting-pods.md

```diff
@@ -0,0 +1,70 @@
+---
+assignees:
+- bprashanth
+- erictune
+- foxish
+- smarterclayton
+
+---
+
+{% capture overview %}
+This page shows how to delete pods which are part of a stateful set, and explains the various considerations when doing so.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* If you are not using Stateful Sets, you can skip this task now. 
+* This is a fairly advanced task and has the potential to violate some properties provided by Stateful Sets.
+* If you are attempting this, you must be well aware of the various considerations enumerated below, and have specific domain knowledge of the application(s) running using Stateful Set.
+
+{% endcapture %}
+
+{% capture steps %}
+
+
+### Stateful Set considerations
+
+In normal operation of a Stateful Set, there is **never** a need to force delete a Stateful Set pod. The Stateful Set controller is responsible for creating, scaling and deleting members of the Stateful Set. It tries to ensure that the specified number of Pods from ordinal 0 through N-1 are alive and ready. Stateful Set also gives us the property that there is at any time at most one pod with a given identity running in a cluster. This is  referred to as *at most one* semantics provided by a Stateful Set. (*TODO: Link to concept page.*)
```

Comment:
Avoid first-person collective constructs ("we", "us"). Instead of "StatefulSet gives us the property", try "StatefulSet ensures that, at any time, there is at most one pod with a given identity running in a cluster."